modules/prebidServerBidAdapter/index.js

```diff
@@ -599,7 +599,17 @@ const OPEN_RTB_PROTOCOL = {
           if (videoParams.context === 'instream' && !videoParams.hasOwnProperty('placement')) {
             videoParams.placement = 1;
           }
-          mediaTypes['video'] = videoParams;
+
+          mediaTypes['video'] = Object.keys(videoParams).filter(param => param !== 'context')
+            .reduce((result, param) => {
+              if (param === 'playerSize') {
+                result.w = utils.deepAccess(videoParams, `${param}.0.0`);
+                result.h = utils.deepAccess(videoParams, `${param}.0.1`);
```

Comment:
why not just hard code it?
```
result.w = videoParams.playerSize[0][0];
result.w = videoParams.playerSize[1][1];
```

Reply:
Also, according to [prebid.org docs](https://docs.prebid.org/prebid-video/video-getting-started.html), `playerSize` could be a one dimensional array `[w, h]` which this doesn't support.

Reply:
It can be hardcoded as well. I only wrote it with deepAccess to be conservative in case of any future changes that could break this. 

playerSize can be defined as a one dimensional array but prebid.js validates and converts playersize into a two dimensional array before sending to bidders